### PR TITLE
Remove more redundant null checks

### PR DESCRIPTION
**Pull request by RealKC**
I missed these last night ^^

### CODE REVIEW

File: gio/src/file_attribute_info.rs [hunk 1]
```diff
@@ -13,30 +13,22 @@ glib::wrapper! {
...
-                (*copy).name = glib::ffi::g_strdup((*ptr).name);
-            }
+            (*copy).name = glib::ffi::g_strdup((*ptr).name);
             copy
         },
         free => |ptr| {
-            if !(*ptr).name.is_null() {
-                glib::ffi::g_free((*ptr).name as *mut _);
-            }
+            glib::ffi::g_free((*ptr).name as *mut _);
```
Comment (RealKC) [inline gio/src/file_attribute_info.rs:20]: Is `ptr` here guaranteed to be nonnull?

Comment (sdroege) [reply gio/src/file_attribute_info.rs:20]: Yes

Comment (RealKC) [reply gio/src/file_attribute_info.rs:20]: That's good then, I looked at the docs and wasn't sure

Comment (sdroege) [reply gio/src/file_attribute_info.rs:20]: It's only called internally in `glib/src/{shared,boxed,boxed_inline,object}.rs` for the corresponding type, and mostly from the `Drop` impl. All the pointers in use there are required to be non-`NULL`.

**Pull request merged**